src/Lesson5/Computer.java

```diff
@@ -0,0 +1,96 @@
+package Lesson5;
+
+import java.util.Random;
+import java.util.Scanner;
+
+/*
+Задание:
+Создать класс компьютер.
+isOn boolean
+- процессор Boolean
+- оперативкаBoolean
+- жесткий диск Boolean
+- ресурс полных циклов работы (включений/выключений) int
+Методы:
+- метод описание(вывод всех полей)
+вывод: [“есть”, “есть”, “есть”, 18 циклов]
+- метод включить, при включении может произойти сбой, при вызове метода
+рандом загадывает число (0 либо 1), вы вводите число с клавиатуры, если
+угадали комп включается, если нет сгорает. Если комп сгорел, при попытке
+включить нужно выдать сообщение что ему конец
+- выключить (аналогично включению)
+- при превышении лимита ресурса комп сгорает
+ */
+public class Computer {
+    static Random random = new Random(); //статичесткие чтобы видеть по всему класу
+    static Scanner scanner = new Scanner(System.in);
+    boolean procesor; //поля екзепляров
+    boolean operativka;
+    boolean zorstkijDisk;
```

Comment:
@Nadya101 
Поменяй названия переменных на без транслитные
```suggestion
    boolean cpu; //поля екзепляров
    boolean ram;
    boolean hdd;
```

Reply:
![Screen HW lesson5](https://user-images.githubusercontent.com/44065623/133587835-c5e9befe-191b-43e2-abf8-88da754bd798.PNG)

Да, я запускала программу перед тем как отправить.